runtime/src/precompiles.rs

```diff
@@ -81,12 +82,17 @@ type PrecompilesAt<R> = (
             CallableByPrecompile<OnlyFrom<AddressU64<2056>>>,
         ),
     >,
+    PrecompileAt<
+        AddressU64<2060>,
+        XcmTeleportPrecompile<crate::Runtime>,
```

Comment:
I think you can use the `R` generic here

Reply:
Yes, done